src/main/java/hw7/jdi/entities/TestData.java

```diff
@@ -0,0 +1,76 @@
+package hw7.jdi.entities;
+
+import java.util.Arrays;
+
+public class TestData {
```

Comment:
What exactly the data ? The name of the class should represent its purpose.

Reply:
Solved: now its name is `MetalsAndColorsFormFiller`